### ISSUE TITLE
Ensure myne.chat is working sufficiently for its dedicated comm week

### ISSUE THREAD

**Issue by thewanderingeditor** [hoprnet/hopr-devrel#209]
### Issue deadline

13th April (myne.chat week will run the week commencing April 18th and comm team would like time to familiarize ourselves with working myne.chat and )

### Detailed Description

As part of the revised Hop on Board plan, we'll be showcasing an alpha version of myne.chat. That goal is that users should be able to easily spin up a local node cluster using gitpod and see how myne.chat looks and works.

### Tasks
- [x] Check that myne.chat still works in light of recent releases and API changes
- [x] Assess the likelihood that myne.chat will continue to work in alpha state for the next 1-3 months without significant upkeep
- [x] Create instructions / make changes needed to make this accessible to less technical community members.
- [x] Test and add Jose's startup tutorial, found in [this branch](https://myne-chat-git-fork-quinszouls-develop-hoprnet.vercel.app/)
- [x] Create instructions to allow multiple users to interface remotely with the same local myne.chat instance.

**Comment by thewanderingeditor** [comment]
Based on a request from Rik, Mynechat would now be showcased in the week starting the 11th. It wouldn't need to be ready exactly on the Monday, but the earlier the better.

**Comment by thewanderingeditor** [comment]
Updated deadline and added new task to reflect requirements for upcoming myne.chat week. Also updated formatting MyneChat -> myne.chat throughout

**Comment by thewanderingeditor** [comment]
Based on 8th April Ignition call, added new task to incorporate [the tutorial on Jose's branch](https://myne-chat-git-fork-quinszouls-develop-hoprnet.vercel.app/)

**Comment by tolbrino** [comment]
> - [ ] Check that myne.chat still works in light of recent releases and API changes

  - Once https://github.com/hoprnet/myne-chat/pull/137 is merged, the Gitpod setup can be used as documented.

> - [ ] Assess the likelihood that myne.chat will continue to work in alpha state for the next 1-3 months without significant upkeep

  - Because Myne Chat is pinned to a stable release it will continue to work even though newer hoprd releases might include breaking changes. This is the case as long as no changes are made in Myne Chat which affect the API usage.

> - [ ] Create instructions / make changes needed to make this accessible to less technical community members.

  - open https://gitpod.io/#https://github.com/hoprnet/myne-chat in a web-browser
  - start the Gitpod session and let the setup run through
  - navigate to the URLs dashboard (3rd list entry in the panel on the bottom right corner)
  - copy full URLs listed as `MyneChat:` into web-browser and start using Myne Chat

> - [ ] Test and add Jose's startup tutorial, found in [this branch](https://myne-chat-git-fork-quinszouls-develop-hoprnet.vercel.app/)
  - that tutorial assumes the user is running hoprd in a testnet where other users can join too, which isn't the case right now

> - [ ] Create instructions to allow multiple users to interface remotely with the same local myne.chat instance.

  - to be done by @Jaguaras

**Comment by tolbrino** [comment]
This tutorial is a good guide for interested developers as well since it explains mostly the inner workings of the Myne Chat Gitpod setup: https://docs.hoprnet.org/developers/starting-local-cluster

**Comment by Jaguaras** [comment]
Instructions for myne-chat has been created #3764.
